triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala

```diff
@@ -224,7 +228,9 @@ trait AbstractTriggerServiceTest
       _ <- mainPackageId should not be empty
       resp <- startTrigger(uri, s"$testPkgId:TestTrigger:trigger", alice)
       triggerId <- parseTriggerId(resp)
+      _ = println("ASSERT TRIGGER IDS")
       _ <- assertTriggerIds(uri, alice, Vector(triggerId))
+      _ = println("STOPPING")
```

Comment:
Debugging?

Reply:
:facepalm: I need a git hook that checks for `println` statements.